docs/settings.sh

```diff
@@ -90,15 +92,3 @@ if [ -z "$DOCS_BUCKET_PRODUCTION" ]; then
   exit 1
 fi
 
-# Sanity check BRANCH and *_BRANCH overrides
-#
-# Note: `compgen` is a bash built-in command and is thus available on
-# all platforms where bash itself is available.
-for branch_variable in BRANCH `compgen -v | grep "_BRANCH$"`; do
-  [[ $branch_variable =~ ^(GIT_BRANCH|CHANGE_BRANCH)$ ]] && continue
-  eval "branch_value=\${$branch_variable}"
-  if [[ "$branch_value" != origin/* && "$branch_value" != v[0-9]* ]]; then
-    echo "ERROR: $branch_variable is set to '$branch_value', which is neither a remote branch ref (e.g. 'origin/3.2.x') nor a tag (e.g. 'v3.2.0')"
-    exit 2
-  fi
-done
```

Comment:
@joel-hamill I had to remove this sanity check because cp-demo has this variable defined: `CONFLUENT_RELEASE_TAG_OR_BRANCH=6.0.0-post`, which this check did not approve of.  I'm not sure what this check accomplishes, but the docs built without it.  LMK if it's critical please